test/DbConnectedResourceAbstractFactoryTest.php

```diff
@@ -91,29 +109,31 @@ public function testWillCreateServiceIfDbConnectedSegmentIsValid($configForDbCon
                 'Foo' => $configForDbConnected,
             ],
         ]];
-        $this->services->set('Config', $config);
-        $this->services->set($tableServiceName, new stdClass());
-        $this->assertTrue($this->factory->canCreateServiceWithName($this->services, 'foo', 'Foo'));
+        $this->services->has('config')->willReturn(true);
+        $this->services->get('config')->willReturn($config);
+
```

Comment:
why the empty line is here? line below belongs to "arrange" tests part, so better it the empty line will be before "assert" part